### PR TITLE
Use the moved newsflash.txt (drop the copy in this repository)

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

**Comment by levitte** [comment]
This is pending an actual move

**Comment by levitte** [comment]
Added a commit that takes care of news/vulnerabilities.xml and
news/secadv/ too, since they're also moving.

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,8 @@
 CHECKOUTS = /var/cache/openssl/checkouts
 ##  Snapshot directory
 SNAP = $(CHECKOUTS)/openssl
+##  OTC data directory
+OMCDATA = $(CHECKOUTS)/data
```
Comment (mattcaswell) [inline Makefile:9]: Comment says OTC, but the variable is `OMCDATA`

**Review by mattcaswell — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (levitte) [reply Makefile:9]: Fixed

**Comment by levitte** [comment]
Ping?

**Review by paulidale — APPROVED** [review]

**Comment by levitte** [comment]
Merged

755faf1f1b459e3266c183025c85881837141474 Use the moved newsflash.txt (drop the copy in this repository)
928bd1ae651b799ffeee005367f7e9268a50c413 Use the moved secadv files too (drop the copies in this repository)

**Comment by levitte** [comment]
I've observed this build properly with our automation, and render expected results.
Mission accomplished :-)

**Pull request closed without merging**